hasher/src/roinput.rs

```diff
@@ -445,11 +446,13 @@ mod tests {
 
     #[test]
     fn all_1() {
-        let scalar1 = Fq::from_hex("01d1755db21c8cd2a9cf5a3436178da3d70f484cd4b4c8834b799921e7d7a102")
-            .expect("failed to create scalar");
-        let scalar2 = Fq::from_hex("e70187e9b125524489d0433da76fd8287fa652eaebde147b45fa0cd86f171810")
-            .expect("failed to create scalar");
-        
+        let scalar1 =
```

Comment:
As for the commits applying the linter, we should squash this commit the one introducing the actual changes. Formatting and linting should be done after each commit.